apps/ex_nvr/lib/ex_nvr/authorization.ex

```diff
@@ -4,6 +4,7 @@ defmodule ExNVR.Authorization do
   """
   alias ExNVR.Accounts.User
   def authorize(%User{role: :admin}, _resource, _action), do: :ok
+  def authorize(%User{role: :user}, :user, _action), do: {:error, :unauthorized}
```

Comment:
Leave a blank line between the `alias` and the function definition and add a `@spec` to the function.